src/DataTable/filters/CheckboxFilter.jsx

```diff
@@ -27,15 +26,19 @@ function CheckboxFilter({
   return (
     <Form.Group role="group" aria-labelledby={ariaLabel.current}>
       <FormLabel id={ariaLabel.current} className="pgn__checkbox-filter-label">{Header}</FormLabel>
-      {filterChoices.map(({ name, number, value }) => (
-        <LabelledCheckbox
-          id={headerBasedId}
-          key={headerBasedId + name}
-          checked={checkedBoxes.includes(value)}
-          onChange={() => { changeCheckbox(value); }}
-          label={<>{name} {number !== undefined && <Badge variant="light">{number}</Badge>}</>}
-        />
-      ))}
+      <Form.CheckboxSet name={Header}>
+        {filterChoices.map(({ name, number, value }) => (
+          <Form.Checkbox
+            key={`${headerBasedId}${name}`}
+            value={name}
+            checked={checkedBoxes.includes(value)}
+            onChange={() => changeCheckbox(value)}
+            aria-label={name}
+          >
+            {name} {number !== undefined && <Badge className="ml-2" variant="light">{number}</Badge>}
```

Comment:
We should avoid using CSS utility classes in Paragon components as they include `!important`, which makes theming/branding difficult for consumers. Consider defining a new `.pgn__` namespaced class name or perhaps using `<Stack direction="horizontal" gap={2}>` here would make sense?

Reply:
Yes, you are right
I will fix it